webapp/test/e2e/tsconfig.json

```diff
@@ -0,0 +1,7 @@
+{
+  "compilerOptions": {
+    "moduleResolution": "node",
+    "types": ["node", "@openui5/types", "@wdio/globals/types", "@wdio/mocha-framework", "wdio-ui5-service/esm"],
```

Comment:
@openui5/types are not a dependency (@types/openui5 are used)

Reply:
fixed